<map id="pylo.vendors.urllib3.exceptions.URLSchemeUnknown" name="pylo.vendors.urllib3.exceptions.URLSchemeUnknown">
<area shape="rect" id="node1" title=" " alt="" coords="29,259,227,300"/>
<area shape="rect" id="node2" href="$classpylo_1_1vendors_1_1urllib3_1_1exceptions_1_1_location_value_error.html" title=" " alt="" coords="29,169,227,211"/>
<area shape="rect" id="node3" title=" " alt="" coords="5,87,88,114"/>
<area shape="rect" id="node4" href="$classpylo_1_1vendors_1_1urllib3_1_1exceptions_1_1_h_t_t_p_error.html" title=" " alt="" coords="111,80,310,121"/>
<area shape="rect" id="node5" title=" " alt="" coords="171,5,250,32"/>
</map>
